src/lib/coda_lib/coda_lib.ml

```diff
@@ -768,6 +766,7 @@ module Make (Inputs : Inputs_intf) = struct
               Strict_pipe.create Synchronous
             in
             let net_ivar = Ivar.create () in
+            let persistence_buffer_capacity = 30 in
```

Comment:
30 seems a little bit small, but we can tune this as we need to. We should set up something to measure our effeciency when processing diffs in transition_frontier_persistence.